mdoc/src/main/scala/mdoc/internal/cli/MainOps.scala

```diff
@@ -34,7 +39,7 @@ final class MainOps(
     if (settings.noLivereload) ()
     else {
       val livereload = UndertowLiveReload(
-        settings.out.toNIO,
+        settings.out.head.toNIO,
```

Comment:
Just to double-check, we would have validated at this point, and would have been stopped before getting here that `out` isn't empty?

Reply:
Correct, it's still not great to call `.head`. I updated the code to validate here and report an actionable message on how to fix it worst case. I also added a warning when we ignore the tail arguments with intructions on how to LiveReload another output directory.